package.json

```
@@ -1,13 +1,20 @@
 {
   "name": "pejuangkode.com",
-  "version": "0.0.1",
+  "version": "1.0.0",
+  "description": "Website pejuangkode.com",
+  "main": "N/A",
+  "repository": "https://github.com/pejuang-kode/pejuangkode.com",
+  "bugs": "https://github.com/pejuang-kode/pejuangkode.com/issues",
+  "license": "MIT",
+  "private": true,
```

Karena ini repository publik, dibuat `false` aja Mas. 😁 
```suggestion
  "private": false,
```